transaction-status/src/lib.rs

```diff
@@ -566,54 +546,101 @@ impl From<EncodedConfirmedBlock> for UiConfirmedBlock {
 }
 
 #[derive(Clone, Debug, PartialEq)]
-pub struct VersionedTransactionWithStatusMeta {
-    pub transaction: VersionedTransaction,
+#[allow(clippy::large_enum_variant)]
+pub enum TransactionWithStatusMeta {
+    // Very old transactions may be missing metadata
+    MissingMetadata(Transaction),
+    // Versioned stored transaction always have metadata
+    Ok(VersionedTransactionWithStatusMeta),
```

Comment:
```suggestion
    Complete(VersionedTransactionWithStatusMeta),
```
...or similar, to disambiguate from the result of the transaction

Reply:
Done